howto/install-appliance/enable-oidc.md

```diff
@@ -0,0 +1,55 @@
+(howto-appliance-enable-oidc)=
+# How to enable custom identity provider support
+
+The Anbox Cloud Appliance has support for custom identity providers for authentication through the use of [OpenID Connect](https://openid.net/developers/discover-openid-and-openid-connect/).
+
+Support for a custom identity provider has to be enabled at initialisation time by using a preseed configuration. See {ref}`ref-appliance-preseed-config` for more details.
+
+Anbox Cloud uses the [Authorisation code flow](https://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth) to obtain an identity token. No access token is being requested as authorisation is handled within the Anbox Cloud services.
```

Comment:
```suggestion
Anbox Cloud uses the [authorisation code flow](https://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth) to obtain an identity token. No access token is requested in this flow because authorisation is handled within the Anbox Cloud services.
```